.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: Generator tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    container: golang:1.13
+    steps:
+      - uses: actions/checkout@v2
+      - run: go test -p 1 ./...
+  bazel-build:
+    runs-on: ubuntu-latest
+    container: gcr.io/gapic-images/googleapis-bazel:20210105
+    # Dockerfile for this image: https://github.com/googleapis/googleapis-discovery/blob/master/Dockerfile
```

Comment:
Maybe we should move to the Dockerfile to a more central place, not one that's tied to `googleapis-discovery` (That's outside the scope of this PR; just suggesting this)

Reply:
cc @alexander-fenster 

Yeah maybe so. We might start using it more broadly if generator repos run bazel builds.